test/Project.toml

```
@@ -44,6 +45,7 @@ ECOS = "1.1.2"
 ExplicitImports = "1.0.1"
 ForwardDiff = "0.10.36, 1"
 LinearAlgebra = "1"
+LinearSolve = "2.32, 3"
```

```suggestion
LinearSolve = "2.35, 3"
```
Required to get EnzymeCore.jl v8, which is needed by CUDA.jl v5.8.